tests/test_simulation.py

```diff
@@ -2071,6 +2093,7 @@ def test_stress_skin2(self, frame_modal_simulation: post.ModalMechanicalSimulati
 
     def test_strain_skin(self, frame_modal_simulation: post.ModalMechanicalSimulation):
         if frame_modal_simulation._model._server.meet_version("7.1"):
+            # todo I think this should be strain instead of stress
             result = frame_modal_simulation.stress_principal_elemental(
```

Comment:
TBD: There  are some tests labeled test_strain_skin, but they actually test stress results. I think this should be changed.